pkg/scripts/os_amzn.go

```diff
@@ -179,7 +178,7 @@ sudo systemctl restart kubelet
 	removeBinariesAmazonLinuxScriptTemplate = `
 sudo systemctl stop kubelet || true
 
-sudo yum versionlock delete kubelet kubeadm kubectl kubernetes-cni cri-tools || true
```

Comment:
The reason why we have this `|| true` is that `yum versionlock delete` fails if there's no versionlock set on packages. I think it makes sense to leave it in place. Is there any concrete reason why are we removing this?

Reply:
Turns out, `yum versionlock delete` do not fail, even if packages do not exist :shrug:.

![image](https://github.com/user-attachments/assets/121ca8aa-5b70-4d40-8eff-b86d0886b601)